src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -1414,13 +1397,6 @@ def load_arguments(self, _):
         c.argument('marker', arg_type=marker_type)
         c.argument('show_next_marker', action='store_true', help='Show nextMarker in result when specified.')
 
-    with self.argument_context('sig share enable-community') as c:
-        c.argument('gallery_name', type=str, help='The name of the Shared Image Gallery.', id_part='name')
-        c.argument('subscription_ids', nargs='+', help='A list of subscription ids to share the gallery.')
-        c.argument('tenant_ids', nargs='+', help='A list of tenant ids to share the gallery.')
```

Comment:
It seems these two params are removed from the `sig share enable-community` command and not added back in the new code implementation. Does it meet your expectations?

Reply:
`sig share` commands call the same url, but for `sig share enable-community` these two params are not required and not handled, so I didn't add them

Reply:
you mean these two params are not used at all in `sig share enable-community` command, right?